test/babashka/neil/dep_upgrade_test.clj

```diff
@@ -0,0 +1,34 @@
+(ns babashka.neil.dep-upgrade-test
+  (:require
+   [babashka.fs :as fs]
+   [babashka.process :refer [tokenize]]
+   [babashka.tasks :as tasks]
+   [clojure.edn :as edn]
+   [clojure.string :as str]
+   [clojure.test :as t :refer [deftest is testing]]))
+
+(defn test-file [name]
+  (doto (fs/file (fs/temp-dir) "neil" name)
+    (-> fs/parent (fs/create-dirs))
+    (fs/delete-on-exit)))
+
+(defn neil [arg & args]
```

Comment:
We can use `babashka.neil.test-util/neil` here so we don't have to duplicate this in each test namespace.

Reply:
Good catch, thanks! Fixed.